docs/src/contribute/contribute-core-rule.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Contribute to Core Rules
+eleventyNavigation:
+    key: contribute core rule
+    parent: contribute to eslint
+    title: Contribute to Core Rules
+    order: 10
+---
+
+The ESLint core rules are the rules included in the ESLint package.
+
+## Rule Writing Documentation
+
+For full reference information on writing rules, refer to [Custom Rules](../extend/custom-rules). Both custom rules and core rules have the same structure and API. The primary difference between core and custom rules are:
+
+1. Core rules are included in the `eslint` package.
+1. Core rules must adhere to the structure documented on this page.
+
+## Core Rule Structure
+
+Each core rule in ESLint has three files named with its identifier (for example, `no-extra-semi`).
+
+* in the `lib/rules` directory: a source file (for example, `no-extra-semi.js`)
+* in the `tests/lib/rules` directory: a test file (for example, `no-extra-semi.js`)
+* in the `docs/src/rules` directory: a Markdown documentation file (for example, `no-extra-semi.md`)
+
+**Important:** If you submit a **core** rule to the ESLint repository, you **must** follow some conventions explained below.
```

Comment:
You can't submit non-core rules to the ESlint repo, so I don't think we need the emphasis
```suggestion
**Important:** If you submit a core rule to the ESLint repository, you **must** follow some conventions explained below.
```